pkgs/applications/office/libreoffice/default.nix

```diff
@@ -364,6 +364,11 @@ in
     ln -s $out/bin/soffice $out/bin/libreoffice
     ln -s $out/lib/libreoffice/share/xdg $out/share/applications
 
+    for f in $out/share/applications/*.desktop; do
+      substituteInPlace "$f" \
+        --replace "Exec=libreoffice${major}.${minor}"    "Exec=$out/bin/soffice"
```

Comment:
absolute paths in Exec= are problematic, see https://github.com/NixOS/nixpkgs/pull/68035

Reply:
wow, that is just another rabbit hole..I guess the easiest thing is to leave it at `"Exec=libreoffice"` then..

Reply:
Thanks @symphorien, good info!

I've put `Exec` back to using `libreoffice`.